News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewFeedActivity.java

```diff
@@ -164,13 +162,8 @@ public void importOpml() {
     }
 
     private void openFilePicker() {
-        new MaterialFilePicker()
-                .withActivity(this)
-                .withRequestCode(REQUEST_CODE_OPML_IMPORT)
-                //.withFilter(Pattern.compile(".*\\.opml$")) // Filtering files and directories by file name using regexp
-                .withFilterDirectories(true) // Set directories filterable (false by default)
-                .withHiddenFiles(true) // Show hidden files and folders
-                .start();
+        startActivityForResult(new Intent(Intent.ACTION_GET_CONTENT)
+                .addCategory(Intent.CATEGORY_OPENABLE).setType("*/*"), REQUEST_CODE_OPML_IMPORT);
```

Comment:
@stefan-niedermann fyi - I had to use `*/*` as .opml files are not always consideres `text/*`. At least my phone didn't recognize them when I used the `text/*` version

Reply:
just switch back to `*/*` then :)